fineract-provider/src/main/java/org/apache/fineract/notification/config/MessagingConfiguration.java

```diff
@@ -35,6 +36,7 @@
 
 @Configuration
 @Profile("activeMqEnabled")
+@ConditionalOnExpression("${fineract.mode.write-enabled:true} || ${fineract.mode.batch-enabled:true}")
```

Comment:
I wish we could use programmatic conditions here with the FineractModeProperties POJO.

Before merging, please look at this: https://stackoverflow.com/a/69901997/2863995
And let's try to make it work.

@taskain7 already created a programmatic Condition here: `FineractModeValidationCondition`

Reply:
Done! Condition created: `EnableFineractEventsCondition`